test/server/cards/02_SHD/units/GreySquadronYWing.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Grey squadron Y-Wing', function() {
+    integration(function(contextRef) {
+        describe('Grey squadron Y-Wing\'s Ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['grey-squadron-ywing'],
+                        groundArena: ['wampa'],
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['cartel-spacer'],
+                    }
+                });
+            });
+
+            it('should deal damage to either a base or a unit (depending on opponent choice)', function () {
+                const { context } = contextRef;
+                context.player1.clickCard(context.greySquadronYwing);
+                context.player1.clickCard(context.cartelSpacer);
+                expect(context.player1).toHaveExactPromptButtons(['Deal 2 damage to an opponent\s base or unit they control (they choose which)', 'Pass']);
+                context.player1.clickPrompt('Deal 2 damage to an opponent\s base or unit they control (they choose which)');
+                expect(context.player2).toHavePrompt('Choose a card');
```

Comment:
Don't need to check the prompt name if there's card selection happening, just confirming the target list is enough